README.md

```diff
@@ -2,6 +2,8 @@
 
 No code is the best way to write secure and reliable applications. Write nothing; deploy nowhere.
 
+CI Status: ![none](https://github.com/kelseyhightower/nocode/actions/workflows/no-op.yml/badge.svg)
+
```

Comment:
@kevtechXx  I like this minimalistic approach with just having

`CI Status:`

This might also improve on the security concerns mentioned in the beginning of the `README.md`.

Reply:
well a non-existent build can't fail